templates/openshift-prune-cronjob.yml.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: build-pruner
+spec:
+  schedule: "{{ osbs_prune_schedule }}"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+          - name: token
+            secret:
+              defaultMode: 420
+              secretName: {{ osbs_prune_secret }}
+          containers:
+          - name: build-pruner
+            image: {{ osbs_prune_image }}
+            command: ["/prune.sh"]
```

Comment:
Can't the default command on the image be used?

Reply:
If buildroot is used it cannot be a default command. Instead this should be parametrized, updated the PR